product_supplierinfo_for_customer/views/product_view.xml

```diff
@@ -95,18 +69,25 @@
         </field>
     </record>
 
-    <record model="ir.ui.view" id="product_template_extended_form_view">
-        <field name="name">product.template.extended.form</field>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.template.common.form</field>
         <field name="model">product.template</field>
         <field name="inherit_id" ref="product.product_template_form_view"/>
         <field name="arch" type="xml">
             <div name="pricelist_item" position="after">
-                <separator string="Customers" />
-                <field name="customer_ids"
-                       context="{'default_search_type':'customer','default_type':'customer','default_product_tmpl_id':id}">
-                </field>
+                <div name="pricelist_customer">
+                    <separator string="Customers" />
+                    <field name="customer_ids"
+                           context="{'default_search_type':'customer','default_type':'customer','default_product_tmpl_id':id,'form_view_ref':'product_supplierinfo_customer_form_view'}">
+                    </field>
+                </div>
             </div>
         </field>
     </record>
 
+    <record id="product.product_supplierinfo_type_action" model="ir.actions.act_window">
+        <field name="name">Partner Pricelists</field>
+        <field name="res_model">product.supplierinfo</field>
```

Comment:
why is this needed?

Reply:
to change name of "Vendor Pricelists" to "Partner Pricelists"

Also, I now updated it to pass the supplier domain

Reply:
ok, now it's good